tests/contrib/test_algorithms.py

```diff
@@ -2,7 +2,7 @@
 
 import pytest
 
-from ..utils import ensure_bytes, ensure_unicode, key_path
+from tests.utils import ensure_bytes, ensure_unicode, key_path
```

Comment:
The relative import makes more sense here to me unless you have a strong justification for the change.